edk2basetools/GenCrc32/GenCrc32.py

```diff
@@ -0,0 +1,103 @@
+## @file
+# Calculate Crc32 value and Verify Crc32 value for input data.
+#
+# Copyright (c) 2007 - 2018, Intel Corporation. All rights reserved.<BR>
```

Comment:
If this is new code, it should probably be "Copyright (c) 2024...." since it wasn't written in 2007.